Clients/GUI/Scripts/BotRunner.cs

```diff
@@ -0,0 +1,29 @@
+﻿using DTO;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using System.Threading.Tasks;
+
+namespace GUI.Scripts
+{
+    public class BotRunner
```

Comment:
I suggest moving all these to class like "BotManager", otherwise too many classes incoming

Reply:
done